### PR TITLE
Plot postfit shapes

### PR DESCRIPTION
**Pull request by Lara813**
Modified task to plot postfit distributions

### CODE REVIEW

File: hbw/ml/mixins.py [hunk 1]
```diff
@@ -97,16 +97,24 @@ def prepare_ml_model(
...
+            )
+        else:
+            model.compile(
+                # NOTE: we'd preferrably use the Keras CCE, but it does not work when assigning one event
+                #       to multiple classes (target with multiple entries != 0)
+                loss=cumulated_crossentropy,
+                optimizer=optimizer,
+                metrics=["categorical_accuracy"],
+                weighted_metrics=["categorical_accuracy"],
+            )
```
Comment (mafrahm) [inline hbw/ml/mixins.py:117]:
```suggestion
        model_compile_kwargs = {
            "loss": "categorical_crossentropy" if self.negative_weights == "ignore" else cumulated_crossentropy,
            "optimizer": optimizer,
            "metrics": ["categorical_accuracy"],
            "weighted_metrics": ["categorical_accuracy"],
        }
        model.compile(**model_compile_kwargs)

File: hbw/ml/data_loader.py [hunk 2]
```diff
@@ -167,6 +179,10 @@ def parameters(self):
         }
         return self._parameters
 
+    @property
+    def skip_mask(self):
+        return self._skip_mask
```
Comment (mafrahm) [inline hbw/ml/data_loader.py:184]: can you add the strings ("num_events_per_process" etc.) as properties instead of this boolean to get rid of all the Fallunterscheidungen?

**Review by mafrahm — CHANGES_REQUESTED** [review]

File: hbw/ml/data_loader.py (hunk 2 above)
Comment (Lara813) [reply hbw/ml/data_loader.py:183]: Good call, I did

File: hbw/ml/tf_util.py [hunk 3]
```diff
@@ -165,6 +165,11 @@ def _cumulated_crossenropy_from_logits(y_true, y_pred, axis):
     return ((numerator - denominator) / tf.math.reduce_sum(y_true, axis=axis))[:, 0]
 
 
+@tf.function
+def categorical_crossentropy():
```
[the remark this answers is not in the document]
Comment (Lara813) [reply hbw/ml/tf_util.py:169]: deleted

**Review by mafrahm — APPROVED** [review]
feini
